designs/simon/src/options.mjs

```diff
@@ -35,6 +35,7 @@ export const collarBend = { pct: 3.5, min: 0, max: 10, menu: 'style.collar' }
 export const collarFlare = { deg: 3.5, min: 0, max: 10, menu: 'style.collar' }
 export const collarGap = { pct: 2.5, min: 0, max: 6, menu: 'style.collar' }
 export const collarRoll = { pct: 3, min: 0, max: 6, menu: 'style.collar' }
+export const collarWidth = { pct: 120, min: 10, max: 300, menu: 'style.collar' }
```

Comment:
Seeing as the timeline for out-of-range values is "TBD" and "contingent on several major changes", I felt it best not to make decisions based on it, and thus didn't want to artificially limit users too much, since the pattern adapts gracefully either way. I'm still down for narrowing the range, if desired.

Reply:
After sleeping on this, I agree with the smaller range being more sensible. Ultimately, the limits are also part of "our" communication with the user, and having such a broad range makes that communication less clear.